### PR TITLE
Create nightly-build-example-tests-ppml.yaml

### PR DESCRIPTION
**Pull request by Romanticoseu**
## Description

Create nightly-build-example-tests-ppml.yaml

### CODE REVIEW

**Comment by liu-shaojun** [comment]
Have you tested if this workflow can run successfully? if not, please test it before merge.

**Comment by Romanticoseu** [comment]
> Have you tested if this workflow can run successfully? if not, please test it before merge.
Tested in prevent pr 5455

**Review by liu-shaojun — APPROVED** [review]

File: .github/workflows/nightly-build-example-tests-ppml.yaml [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+        --device=/dev/sgx/enclave \
+        --device=/dev/sgx/provision \
+        -v ~/glorysdj/kuberconfig:/root/.kube/config \
+        -v /var/run/aesmd/aesm.socket:/var/run/aesmd/aesm.socket \
+        -v $ENCLAVE_KEY:/graphene/Pal/src/host/Linux-SGX/signer/enclave-key.pem \
+        -v $DATA_PATH:/ppml/trusted-big-data-ml/work/data \
+        -v $KEYS_PATH:/ppml/trusted-big-data-ml/work/keys \
+        -e LOCAL_IP=$LOCAL_IP \
+        -e SGX_MEM_SIZE=$SGX_MEM_SIZE \
+        -e NotebookPort=12345 \
```
Comment (glorysdj) [inline .github/workflows/nightly-build-example-tests-ppml.yaml:68]: NotebookPort NotebookToken can be removed

File: .github/workflows/nightly-build-example-tests-ppml.yaml [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+        -e NotebookToken="1234qwer" \
+        -e RUNTIME_SPARK_MASTER=k8s://https://192.168.0.112:6443 \
```
Comment (glorysdj) [inline .github/workflows/nightly-build-example-tests-ppml.yaml:70]: k8s master can be set as input with default value k8s://https://192.168.0.112:6443

**Pull request merged**